control-plane-operator/controllers/hostedcontrolplane/pki/node-tuning-operator.go

```diff
@@ -0,0 +1,21 @@
+package pki
+
+import (
+	"fmt"
+
+	"github.com/openshift/hypershift/support/config"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// Create TLS keys for ppc-webhook-cert.
+// NOTE: PPC stands for PerformanceProfileController. We use ppc in here to keep it short and avoid
+//
+//	possible problems with kubernetes name limits.
+func ReconcilePerformanceProfileControllerWebhook(secret, ca *corev1.Secret, ownerRef config.OwnerRef) error {
```

Comment:
If we're not creating a ValidatingWebhookConfiguration inside the hosted cluster, then I don't see a need to create a cert for it. 

From a user's point of view, I don't think it makes sense to have a validating webhook in the guest cluster, since performance profiles are driven by whatever is specified in the NodePool. It looks like we already validate the resource here: https://github.com/openshift/hypershift/pull/1782/commits/5080bd7dddf53eb7c1c7254e5d6166fe59f9c542#diff-c666a2a9dc48b0d3de33cc8fa3a054f853671f0428f9d56b263750dfc6d26e00R2018
If there are any additional validations to be made, then I would add them there.

Reply:
Totally agree.
Thanks for the comment.
I have deleted the webhook configuration [here](https://github.com/openshift/hypershift/pull/1782/commits/dcfb5a5091611d8b5e5f63d925a749432f4e43a6) ( did it in a different commit just for clarity but will be squashed with the previous one before merge)